mobile/function/follow_location.js

```diff
@@ -0,0 +1,111 @@
+import { useEffect, useState } from "react";
+import * as TaskManager from "expo-task-manager";
+import * as Location from "expo-location";
```

Comment:
Horen die sterretjes, lijkt mij dat je dingen importeerd die je niet gebruikt maar idk

Reply:
Dit is naar mijn idee wel nodig